src/util/DebouncedOnChange/index.tsx

```diff
@@ -18,13 +18,25 @@ export interface DebouncedOnChangeProps {
    * milliseconds.
    */
   debouncedOnChangeTime?: number;
+
+  /**
+   * Optional callback to access the component interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?(a: any): void;
 }
 
 export interface DebouncedOnChangePrivateProps {
   /**
-   * Used to pass both onChange and debouncedOnChange to the contained component.
+   * Private: Used to pass both onChange and debouncedOnChange to the contained component.
```

Comment:
The whole interface is pretty clear about it's private status, maybe we don't need to reiterate in the description?